pymodm/fields.py

```diff
@@ -778,6 +778,12 @@ def __init__(self, field=None, verbose_name=None, mongo_name=None,
         super(ListField, self).__init__(verbose_name=verbose_name,
                                         mongo_name=mongo_name,
                                         **kwargs)
+
+        if not isinstance(field, MongoBaseField):
+            raise ValueError(
+                'field must be an instance of MongoBaseField, not %s' % (field,)
+            )
```

Comment:
Can you put these trailing parens on the line above? That's the convention we usually follow in the rest of the codebase.